tests/test_pod_exec.py

```diff
@@ -27,3 +42,8 @@ def main():
     if err:
         print(err)
     print(ret)
+
+def run(**kwargs):
+
+  output = run_command_on_pod(cmd='ceph osd df')
```

Comment:
I think you want to check the returned value (output) and return 0 based on the return code